### DISCUSSION TITLE
I want to implement snap to grid in React Flow version 11. Do we have any example code in codesandbox ?

### DISCUSSION THREAD

**Discussion by sudarsanarao995** [xyflow/xyflow#3640] [General]
how to implement snap to grip in React Flow, after implement how it will look like in UI

**Comment by bcakmakoglu** [comment]
You can use snap-grid by passing the props `snapToGrid` and `snapGrid` to `ReactFlow`:

```tsx
<ReactFlow
	nodes={nodes}
	edges={edges}
	snapToGrid
	snapGrid={[10, 10]}
	onNodesChange={onNodesChange}
	onEdgesChange={onEdgesChange}
	onConnect={onConnect}
	nodeTypes={nodeTypes}
	fitView
/>
```

[Here's an example.](https://codesandbox.io/s/long-surf-sj7jxm?file=/src/Flow.tsx:1124-1403)
